engines/sci/detection.cpp

```diff
@@ -119,8 +119,8 @@ static const PlainGameDescriptor s_sciGameTitles[] = {
 	{"hoyle4",          "Hoyle Classic Card Games"},
 	{"inndemo",         "ImagiNation Network (INN) Demo"},
 	{"kq6",             "King's Quest VI: Heir Today, Gone Tomorrow"},
-	{"laurabow2",       "Laura Bow 2: The Dagger of Amon Ra"},
-	{"qfg1vga",         "Quest for Glory I: So You Want to Be a Hero"},	// Note: There was also a SCI0 version of this (further up)
+	{"laurabow2",       "Roberta Williams' Laura Bow in The Dagger of Amon Ra"}, //also called Laura Bow II
```

Comment:
The most widely used naming for this game is just "The Dagger of Amon Ra".
Personally I'd go with "Laura Bow in: The Dagger of Amon Ra" or use Sierra Collection Series naming and also change The Colonel's Bequest:

Laura Bow I: The Colonel's Bequest
Laura Bow II: The Dagger of Amon Ra

Reply:
I agree with this Ill change it back and to this